cluster/gce/trusty/kube-manifests/kube-apiserver.manifest

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
The kube-apiserver and kube-constroller-manager manifest in cluster/saltbase/salt/ have a couple of volumeMounts and volumes that does not work in our images. So, I have to remove them and make separate copies under cluster/gce/trusty/kube-manifests. For all other manifests, I directly copy them from cluster/saltbase/salt.